.github/workflows/stale.yaml

```
@@ -28,8 +32,12 @@ jobs:
 
           # Pull requests configuration
           stale-pr-message: |
-            This pull request has been automatically marked as stale because it has not had 
-            any activity in the last 6 months. It will be closed if there is no further activity.
+            This pull request has been automatically marked as stale because it has not had any activity in the last 6 months. 
+            It will be closed in 30 days if there is no further activity.
+            
+            If the pull request is waiting for a reply from reviewers, feel free to ping them as a reminder.
+            This will also reset the pull request's stale state.
```

```suggestion
            If the pull request is waiting for a reply from reviewers, feel free to ping them as a reminder.
            If it is waiting and has no assigned reviewer, feel free to ping any Spack developer or simply leave a comment saying this should not be marked stale.
            This will reset the pull request's stale state.
```
And I'd suggest similar for the issue message